File: JavaLab/src/main/java/ch3/section2_method/Quiz1.java
package ch3.section2_method;

class Math {
    int plus(int no1, int no2){
        return no1 + no2;
    }
    int plus(int no1, int no2, int no3){
        return no1 + no2 + no3;
    }
    int plus(int[] array){
        int sum = 0;
        for(int no : array){
            sum += no;
        }
        return sum;
    }
}

public class Quiz1 {
    public static void main(String[] args) {
        Math m = new Math();
        System.out.println(m.plus(10, 20));
        System.out.println(m.plus(10, 20, 30));
        System.out.println(m.plus(new int[]{10, 20, 30, 40, 50}));
    }
}

File: KotlinLab/KotlinLab/src/main/kotlin/ch2/section3_function/q3/Quiz1.kt
package ch2.section3_function

class Math {
    fun plus(): Int = 0
    fun plus(no1: Int): Int = no1
    fun plus(no1: Int, no2: Int): Int = no1 + no2
    fun plus(no1: Int, no2: Int, no3: Int): Int = no1 + no2 + no3
}

fun main() {
    val m = Math()
    println(m.plus())
    println(m.plus(10))
    println(m.plus(10, 20))
    println(m.plus(10, 20, 30))
}